server/utils/routes.js

```diff
@@ -9,10 +9,18 @@ export const routes = {
   login: `/login`,
   signup: `/signup`,
   logout: `/logout`,
+  createdConfiguration: (_id) => `${routes.root}u/configure?s=edit&id=${_id}`,
 }
 
 export const v1Routes = {
   adminRoutes: {
     updateUser: `${v1Root}/admin/users/:uid`,
   },
+  configRoutes: {
```

Comment:
I think the `Routes` suffix is implied. The name of the constant is `v1Routes` so we would be duplicating it everywhere.

```suggestion
  user_configurations: {
```